src/gate_detect/src/GateDetectorNode.cpp

```diff
@@ -42,14 +68,20 @@ const sensor_msgs::ImageConstPtr& msg) {
         return;
     }
 
-    lineImg = cv_ptr->image;
+    cv::Mat image = (cv_ptr->image);
+
+    lineImg = cv::Mat(image,
+                      cv::Rect((image.cols / 2) - (width_ / 2),
+                               (image.rows / 2) - (height_ / 2),
+                               width_,
+                               height_));
 
     GateCoordinates gateCoordinates = gateDetector_.initialize(lineImg);
 
     publishGateDetectMsg(gateCoordinates);
 
     // Uncomment if you want to view gate seen by node
-    publishGateImage(gateCoordinates);
+    // publishGateImage(gateCoordinates);
```

Comment:
Can we make a `bool` rosparam like `display_filtered_gate` so that we can enable/disable this without having to recompile? (And also so that it's obvious to people in the future that this is an option)

Reply:
added a bool rosparam